tsstats/templates/index.jinja2

```diff
@@ -24,23 +27,30 @@
   </style>
 </head>
 <body>
-<div class="container-fluid">
-  <nav class="navbar navbar-toggleable-md navbar-light bg-faded sticky-top">
-    <a class="navbar-brand" href="#">{{ title }}</a>
+<nav class="navbar navbar-toggleable-md navbar-light bg-faded sticky-top">
+  <div class="d-flex justify-content-between hidden-lg-up">
+    <a href="" class="navbar-brand">{{ title }}</a>
+    <button class="navbar-toggler hidden-lg-up" type="button" data-toggle="collapse" data-target="main-nav" aria-controls="main-nav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+  </div>
 
-    <div class="collapse navbar-collapse">
-      <ul class="navbar-nav mr-auto">
-        {% for sid, _ in servers %}
-          <li class="nav-item">
-            <a class="nav-link" href="#sid{{ sid }}">Server {{ sid }}</a>
-          </li>
-        {% endfor %}
-      </ul>
-      {% if debug %}
-      <span class="navbar-text" style="color: red">debug mode</span>
-      {% endif %}
-    </div>
-  </nav>
+  <div id="main-nav" class="collapse navbar-collapse">
+    <ul class="navbar-nav mr-auto">
+      <li class="hidden-md-down"><a href="" class="navbar-brand">{{ title }}</a></li>
```

Comment:
One is used for mobile and tablet, the other is used for desktop. The new navbar markup means it's much easier to do this than to try and only have one `navbar-brand` element.